src/components/call_out/call_out.tsx

```diff
@@ -35,6 +37,8 @@ export type EuiCallOutProps = CommonProps &
     color?: Color;
     size?: Size;
     heading?: Heading;
+    onClose?: () => void;
+    isDismissible?: boolean;
```

Comment:
Instead of having two different props, what do you think about having a single prop that determines whether the button renders? If `onClose` is passed, the button will show. This is useful because it prevents the scenario where a consumer passes one prop without the other and you end up with a button that does nothing on close.

```suggestion
    onClose?: () => void;
```

Reply:
Even better, thanks @breehall!